src/material-experimental/mdc-list/list.scss

```diff
@@ -8,3 +8,53 @@
 .mat-mdc-list-base {
   display: block;
 }
+
+// While MDC expects only span elements under mdc-list-item__text, we have historically supported
```

Comment:
Do they have any examples where there's a header etc. _outside_ `mdc-list-item__text`?

Reply:
They do not: https://material-components.github.io/material-components-web-catalog/#/component/list